packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -68,7 +68,7 @@ if (DEV_MODE) {
       password: functionConfig.db.pass,
       database: functionConfig.db.name,
       host: `/cloudsql/${functionConfig.db.host}`,
-      ssl: !DEV_MODE,
+      ssl: functionConfig.db.ssl,
```

Comment:
Do we need to parse this to boolean like we do for the signer config?

Reply:
good point, tested in staging with/out bool parsing and confirmed that it's needed.